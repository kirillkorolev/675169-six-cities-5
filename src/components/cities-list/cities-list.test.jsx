import React from "react";
import renderer from "react-test-renderer";
import CitiesList from "./cities-list";

it(`Should FavoriteEmpty render correctly`, () => {
  const citiesNames = [`Paris`, `London`, `Moscow`];
  const city = [`Moscow`];

  const tree = renderer
    .create(
        <CitiesList
          citiesNames={citiesNames}
          city={city}
          changeCity={() => {}}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
