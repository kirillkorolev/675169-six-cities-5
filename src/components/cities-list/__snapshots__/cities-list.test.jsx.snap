// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FavoriteEmpty render correctly 1`] = `
Array [
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item   "
      href="#"
      onClick={[Function]}
    >
      <span>
        Paris
      </span>
    </a>
  </li>,
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item   "
      href="#"
      onClick={[Function]}
    >
      <span>
        London
      </span>
    </a>
  </li>,
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item   "
      href="#"
      onClick={[Function]}
    >
      <span>
        Moscow
      </span>
    </a>
  </li>,
]
`;
